### PR TITLE
created the covid jumbo without the broken merge

### PR DESCRIPTION
**Pull request by Elena-alexeenko**

### CODE REVIEW

File: new-style.css [hunk 1]
```diff
@@ -726,6 +726,51 @@ html[dir="rtl"] .footer .navs {
...
+    width: 9.25em;
+    height: 9.25em;
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    align-items: flex-end;
+  }
+  
+html[dir='rtl'] .covid .imageCovid {
+    width: 9.25em;
```
Comment (soryy708) [inline new-style.css]: You're not changing `width`, `height`, `align-items` and `bottom` so why repeat them?

File: new-style.css [hunk 2]
```diff
@@ -726,6 +726,51 @@ html[dir="rtl"] .footer .navs {
...
+    height: 9.25em;
+    right:initial;
+    left: 0;
+    bottom: 0;
+    align-items: flex-end;
+    transform: scale(-1,1);
```
Comment (soryy708) [inline new-style.css]: Why not `scaleX(-1)`?

File: new-style.css [hunk 3]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
     .new-steps .step .description {
         font-size: 0.875em;
     }
+
+    /*Covid*/
+
+    .covid .covid-content .covid-container {
+        display: flex;
```
Comment (soryy708) [inline new-style.css]: It's already `display:flex` so why repeat it?

File: new-style.css [hunk 4]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+      }
+    
+      .covid .covid-content .covid-container p {
+        color:#fff;
```
Comment (soryy708) [inline new-style.css]: It's already white so why repeat it?

File: new-style.css [hunk 5]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        text-align: start;
```
Comment (soryy708) [inline new-style.css]: Isn't it already aligned to the start?

File: new-style.css [hunk 6]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        padding-left: 0.75em;
+        padding-right: 0.75em;
+      }
+    
+      .covid .covid-content h1 {
+        color:#fff;
```
Comment (soryy708) [inline new-style.css]: It's already white so why repeat it?

File: new-style.css [hunk 7]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        text-align: center ;
+        padding-inline-start: 0.25em;
+        padding-inline-end: 0.75em;
+      }
+    
+      html[dir="rtl"] .covid .covid-content h1 {
+        color:#fff;
+        right:initial;
+        padding-inline-start: 0.75em;
```
Comment (soryy708) [inline new-style.css]: Why?

File: new-style.css [hunk 8]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        padding-inline-end: 0.75em;
```
Comment (soryy708) [inline new-style.css]: It's already `0.75em` so why repeat it?

File: new-style.css [hunk 9]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+    
+      .covid .covid-content h1 {
+        color:#fff;
+        text-align: center ;
+        padding-inline-start: 0.25em;
+        padding-inline-end: 0.75em;
+      }
+    
+      html[dir="rtl"] .covid .covid-content h1 {
+        color:#fff;
```
Comment (soryy708) [inline new-style.css]: It's already white so why repeat it?

File: new-style.css [hunk 10]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        right:initial;
```
Comment (soryy708) [inline new-style.css]: It's not absolutely positioned so `right` doesn't do anything. Why is it here?

File: new-style.css [hunk 11]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        padding-inline-start: 0.75em;
+        padding-inline-end: 0.75em;
+      }
+    
+      html[dir="rtl"] .covid .covid-content .covid-container p {
+        color:#fff;
```
Comment (soryy708) [inline new-style.css]: It's already white so why repeat it?

File: new-style.css [hunk 12]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        text-align: start;
+        padding-inline-start: 0.75em;
```
Comment (soryy708) [inline new-style.css]: It already has 0.75em padding on both sides so why repeat it?

File: new-style.css (hunk 7 above)
Comment (Elena-alexeenko) [reply new-style.css]: what is problem ?

File: new-style.css [hunk 13]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+      html[dir="rtl"] .covid .covid-content h1 {
+        color:#fff;
+        right:initial;
+        padding-inline-start: 0.75em;
+        padding-inline-end: 0.75em;
+      }
+    
+      html[dir="rtl"] .covid .covid-content .covid-container p {
+        color:#fff;
+        text-align: start;
```
Comment (soryy708) [inline new-style.css]: It's already aligned to start, so why repeat it?

File: new-style.css [hunk 14]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        padding-inline-start: 0.75em;
+        padding-inline-end: 0.75em;
+      }
+    
+      .covid .imageCovid {
+        width: 5em;
+        height: 5em;
+        position: absolute;
```
Comment (soryy708) [inline new-style.css]: It's already absolutely positioned, so why repeat it?

File: new-style.css [hunk 15]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        right: 0;
```
Comment (soryy708) [inline new-style.css]: It's already `right:0` so why repeat it?

File: new-style.css [hunk 16]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+        align-items:flex-end;
```
Comment (soryy708) [inline new-style.css]: It's already `align-items:flex-end` so why repeat it?

File: new-style.css [hunk 17]
```diff
@@ -908,6 +953,55 @@ html[dir="rtl"] .footer .navs {
...
+      }
+    
+      html[dir='rtl'] .covid .imageCovid {
+        width: 5em;
```
Comment (soryy708) [inline new-style.css]: It's already sized to `5em` so why repeat it?

File: new-style.css (hunk 7 above)
Comment (soryy708) [reply new-style.css]: Why Hebrew requires 0.75em padding from the start (right) but English doesn't? (has 0.25em from start (left))

File: new-style.css (hunk 8 above)
Comment (soryy708) [reply new-style.css]: still a thing

File: new-style.css (hunk 13 above)
Comment (soryy708) [reply new-style.css]: still a thing

File: new-style.css (hunk 16 above)
Comment (soryy708) [reply new-style.css]: still a thing

**Pull request merged**